src/NexusMods.DataModel/Loadouts/IngestSteps/BackupFile.cs

```diff
@@ -0,0 +1,30 @@
+using NexusMods.DataModel.Loadouts.ApplySteps;
+using NexusMods.Hashing.xxHash64;
+using NexusMods.Paths;
+
+namespace NexusMods.DataModel.Loadouts.IngestSteps;
+
+/// <summary>
+/// Marks a file as requiring backup.
+/// </summary>
+/// <remarks>
+///    Used when a mod replaces an existing [game] file and the file
+///    is not already backed up.
+/// </remarks>
+public record BackupFile : IIngestStep
+{
+    /// <summary>
+    /// Location of the file to be backed up.
+    /// </summary>
+    public required AbsolutePath To { get; init; }
```

Comment:
This should be renamed to `From` or `Path` since it's the location of the file that's to be backed up. This naming issue is also present in `ApplySteps.BackupFile`, `ApplyStep.DeleteFile` and `ApplyStep.RemoveFromLoadout`.